src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreJob.cs

```diff
@@ -236,6 +240,16 @@ private async Task RestoreAsync(bool forceRestore, RestoreOperationSource restor
                             new SolutionRestoredEventArgs(_status, solutionDirectory));
                     }
                 }
+                catch when (token.IsCancellationRequested)
```

Comment:
probably a better idea to catch OperationCancelledException. 

If there's an actual problem + cancellation later, we should print the actual problem as that's more important to the customer as the `actual problem` is likely to happen again.

Reply:
code changed to catch `OperationCancelledException`